### PR TITLE
Added default table and snapshot container names

### LINKED ISSUE #63: Add default table names
Need to add default table and blob container names for table and snapshot storage, respectively.

### PR DESCRIPTION
**Pull request by IgorFedchenko**
Close #63

### CODE REVIEW

File: src/Akka.Persistence.Azure/Journal/AzureTableStorageJournalSettings.cs [hunk 1]
```diff
@@ -75,10 +79,14 @@ public sealed class AzureTableStorageJournalSettings
         public static AzureTableStorageJournalSettings Create(Config config)
         {
             var connectionString = config.GetString("connection-string");
-            var tableName = config.GetString("table-name");
+            var tableName = config.GetString("table-name", DefaultTableName);
```
Comment (IgorFedchenko) [inline src/Akka.Persistence.Azure/Journal/AzureTableStorageJournalSettings.cs:82]: I realize that easier way would be to set this default names in default hocon configuration - but this way users will be able to get name value from `DefaultTableName` constant in runtime. Which might be useful.

File: src/Akka.Persistence.Azure/Snapshot/AzureBlobSnapshotStoreSettings.cs [hunk 2]
```diff
@@ -61,11 +66,20 @@ public sealed class AzureBlobSnapshotStoreSettings
         public static AzureBlobSnapshotStoreSettings Create(Config config)
         {
             var connectionString = config.GetString("connection-string");
-            var containerName = config.GetString("container-name");
             var connectTimeout = config.GetTimeSpan("connect-timeout", TimeSpan.FromSeconds(3));
             var requestTimeout = config.GetTimeSpan("request-timeout", TimeSpan.FromSeconds(3));
             var verbose = config.GetBoolean("verbose-logging", false);
-            return new AzureBlobSnapshotStoreSettings(connectionString, containerName, connectTimeout, requestTimeout,
+            var containerName = config.GetString("container-name", DefaultContainerName);
```
Comment (IgorFedchenko) [inline src/Akka.Persistence.Azure/Snapshot/AzureBlobSnapshotStoreSettings.cs:72]: Same as above about loading name in runtime.

File: src/Akka.Persistence.Azure/Journal/AzureTableStorageJournalSettings.cs (hunk 1 above)
Comment (Aaronontheweb) [reply src/Akka.Persistence.Azure/Journal/AzureTableStorageJournalSettings.cs:82]: These values should be stored in HOCON.

File: src/Akka.Persistence.Azure/Snapshot/AzureBlobSnapshotStoreSettings.cs (hunk 2 above)
Comment (Aaronontheweb) [reply src/Akka.Persistence.Azure/Snapshot/AzureBlobSnapshotStoreSettings.cs:72]: These values should be stored in HOCON.

**Comment by IgorFedchenko** [comment]
Oops, CI failed

**Comment by IgorFedchenko** [comment]
Should be good to go now

**Pull request merged**